app/src/main/res/values/dimens.xml

```diff
@@ -66,4 +66,10 @@
 
     <!-- Data Sync -->
     <dimen name="pokemon_image_size">200dp</dimen>
+
+
+    <!-- GraphQL -->
+    <dimen name="orders_margin">10dp</dimen>
```

Comment:
It's good since they are even dimens, but is a good practice to use mutiples of 4

Reply:
noted